autoPyTorch/api/base_task.py

```diff
@@ -822,7 +833,8 @@ def _do_traditional_prediction(self, time_left: int, func_eval_time_limit_secs:
                     stats=stats,
                     memory_limit=memory_limit,
                     disable_file_output=self._disable_file_output,
-                    all_supported_metrics=self._all_supported_metrics
+                    all_supported_metrics=self._all_supported_metrics,
+                    evaluator_class=TimeSeriesForecastingTrainEvaluator if self.time_series_forecasting else None,
```

Comment:
Please refer to [this](https://github.com/automl/Auto-PyTorch/pull/434/files#r895920906) comment.